src/internal/m365/collection/exchange/collection.go

```diff
@@ -216,11 +215,11 @@ func (col *prefetchCollection) streamItems(
 	}()
 
 	if len(col.added)+len(col.removed) > 0 {
-		colProgress = observe.CollectionProgress(
+		progressMessage := observe.CollectionProgress(
```

Comment:
nit: `progressMessage` already declared in var block

```suggestion
		progressMessage = observe.CollectionProgress(
```